### PR TITLE
Add Mobile Page Content

### PR DESCRIPTION
**Pull request by Retky**
# Add mobile Main content
## Recent works
- Featured work
- works list

## About
- Me
- Proficiency

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,6 +34,119 @@ <h2 id="enphasis">I’am a Software Developer</h2>
...
+          <img id="featImg" src="img/FeaturedPlaceholder.png" alt="Gym Fit page preview">
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <ul class="languages">
+            <li class="featLang">css</li>
+            <li class="featLang">html</li>
+            <li class="featLang">bootstrap</li>
+            <li class="featLang">ruby</li>
+          </ul>
+          <button id="seeFeat" type="button" name="seeFeat">See Project</button>
```
Comment (mateo951) [inline index.html:51]: Awesome semantics! Well done 😎

File: index.html [hunk 2]
```diff
@@ -33,6 +34,119 @@ <h2 id="enphasis">I’am a Software Developer</h2>
           <li> <a href="#"> <img class="socialIco" src="img/Social_Twitter.png" alt="Twitter logo"> </a> </li>
           <li> <a href="#"> <img class="socialIco" src="img/Social_Medium.png" alt="Medium logo"> </a> </li></ul>
       </section>
+
+      <section id="pageContent">
```
Comment (mateo951) [inline index.html:38]: `display: grid` used splendidly 🙌

File: index.html [hunk 3]
```diff
@@ -33,6 +34,119 @@ <h2 id="enphasis">I’am a Software Developer</h2>
...
+          <ul class="prof">
+            <li id="profSkills"><h3>Skills</h3></li>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </section>
```
Comment (mateo951) [inline index.html:149]: Amazing work 👌

**Review by mateo951 — APPROVED** [review]
### Hi @Retky, 

📝👌 **Your project highlights** 🔥😎 
- Linters are on point ✔
- Good README 🔥
- Great use of GitHub flow 👌

What a fantastic job you did 🔥😎🔥 I didn't find anything to improve in your work ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🧐
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mateo951** [comment]
### OPTIONAL - All of the suggestions inside this comment are optional 👀

You did pretty well writing your summary on your PR's description so this is just an optional/suggestion for your future PRs to improve the structure and flow of it 🙌 

For example, you could go with something like this 🤓
  - Added setup for linters
  - Added gitignore file with ignored files/directories 
  - Added main structure (index.html / style.css)
  - Modified hypothetical.file 😉

Also, you could simplify your title to make your PR clearer such as
**Portfolio: Works & Aboutme**  _(Name of Project: Feature introduced in branch)_

**Comment by Retky** [comment]
Thank you @mateo951
😃

I'll consider your recommendations.
I really appreciate your review.

**Pull request merged**